Gruntfile.js

```diff
@@ -59,7 +58,7 @@ var _              = require('lodash'),
                         'core/client/dist/index.html'
                     ],
                     options: {
-                        livereload: true
+                        livereload: 35729
```

Comment:
Even though this number is the default livereload port - I thought it better to explicitly list it -> see the line comment in TryGhost/Ghost-Admin#177 for more details.